daemon/daemon_windows.go

```diff
@@ -39,9 +39,10 @@ const (
 	windowsMaxCPUPercent = 100
 )
 
-// Windows doesn't really have rlimits.
+// Windows containers are much larger than Linux containers and each of them
+// have > 20 system processes which why it is better to restore them without parallelism.
 func adjustParallelLimit(n int, limit int) int {
```

Comment:
We should change this function to return a `int64` (looks like we always convert the value);

https://github.com/moby/moby/blob/32688a47f34181af671157eff9bc9c5697b321bd/daemon/daemon.go#L223-L227

https://github.com/moby/moby/blob/32688a47f34181af671157eff9bc9c5697b321bd/daemon/daemon.go#L576-L579


(doesn't have to be done in this PR, but just noticed it)

Reply:
I suggest that it will be done on another PR